src/GitLabApiClient/Internal/Http/GitLabApiPagedRequestor.cs

```diff
@@ -15,6 +16,35 @@ internal sealed class GitLabApiPagedRequestor
 
         public GitLabApiPagedRequestor(GitLabApiRequestor requestor) => _requestor = requestor;
 
+        public async Task<(RateLimitPagedInfo rateLimitInfo, IList<T>)> GetRateLimitPagedList<T>(string url, PageOptions pageOptions)
+        {
+            var (results, headers) = await _requestor.GetWithHeaders<IList<T>>(
+                GetPagedUrl(url, pageOptions.Page, pageOptions.ItemsPerPage));
+
+
+            var rateLimitInfo = new RateLimitPagedInfo
```

Comment:
nit: rename variable to `rateLimitPagedInfo` (or `rateLimitPagingInfo` is you rename the class)

Reply:
Updated